src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs

```diff
@@ -8,6 +8,9 @@ namespace SFA.DAS.EmployerUsers.Application.Commands.ActivateUser
 {
     public class ActivateUserCommandValidator : IValidator<ActivateUserCommand>
     {
+        private string _invalidCodeMessage = "Code is not recognised";

```

Comment:
add a const keyword and update the name as appropriate

Reply:
I am generally wary of `const` for things that are not really constant, due to nature of consts. But in this case we would always rebuild whole thing if we changed that string anyhow, so happy to change it.